services/ui-src/src/components/fields/DateField.tsx

```diff
@@ -59,6 +59,9 @@ export const DateField = ({
   // if should autosave, submit field data to database on blur
   const onBlurHandler = async (event: InputChangeEvent) => {
     const { name, value } = event.target;
+    // if blanking field, trigger client-side field validation error
+    if (value === defaultValue) form.trigger(name);
```

Comment:
Note: This makes sure we run the client-side validation on the blurred field if it's being blanked. Without it, the field doesn't get the red warning error when blank it telling you that you've been bad. But we want that error to show up to tell people to not do that. We still let them save it because we revert the field to the default value and then the server-side validation lets it through, but we want the client-side validation to yell at them.

Reply:
_same change across multiple fields_